doc/admin/deployment.md

```diff
@@ -434,10 +434,10 @@ section of the server configuration file for best performance.
 Note that other storage query commands are also available,
 `dmg storage --help` for listings.
 
-SSD health state can be verified via `dmg storage query nvme-health`:
+SSD health state can be verified via `dmg storage scan --nvme-health`:
 
 ```bash
-$ dmg -l wolf-71 storage query nvme-health
+$ dmg -l wolf-71 storage scan --nvme-health
```

Comment:
Why the name is renamed from 'query' to 'scan'?

Reply:
the command was changed so that the health information can be reached from storage scan as it makes better sense from an administrator perspective